### PR TITLE
Completed user sign up functionality for Devise and tests for user model

### PR DESCRIPTION
**Pull request by jhmendes**
Finished user functionality for signing up through devise and all tests.

### CODE REVIEW

**Comment by jhmendes** [comment]
Hey Dan!  Created my first pull request, finished up the functionality for user sign up through Devise. 

I have foundation installed and want to do some base styling but not 100% sure how the foundation files work.  Right now when I run 'rails server' it takes a bit for it to kick in, I think I'm loading in all the foundation components every time.  Any ideas on that or documentation on using Foundation with Rails?  I didn't seem to find much on the Zurb website on how the files are linked up.

File: spec/models/cider_spec.rb [hunk 1]
```diff
@@ -2,4 +2,10 @@
 
 RSpec.describe Cider, type: :model do
   pending "add some examples to (or delete) #{__FILE__}"
+
+  it "is valid with valid attributes"
```
Comment (dpickett) [inline spec/models/cider_spec.rb]: I would probably omit these behaviors and  include them in a separate pull request

File: spec/features/devise/sign_in_spec.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
```
Comment (dpickett) [inline spec/features/devise/sign_in_spec.rb:1]: I would complete this feature spec to round out the testing of devise

**Review by dpickett — APPROVED** [review]
well done

**Pull request merged**